openapi.yml

```diff
@@ -576,6 +581,47 @@ components:
           type: array
           items:
             $ref: "#/components/schemas/DescriptiveValue"
+    DescriptiveParallelContributor:
+      description: Value model for multiple representations of information about the same contributor (e.g. in different languages).
+      type: object
+      additionalProperties: false
+      allOf:
+        - $ref: "#/components/schemas/DescriptiveStructuredValue"
```

Comment:
how is this a DescriptiveStructuredValue and not a DescriptiveStructuredContributor?  Is there a use case that would show this is required?

Reply:
I was modeling it on DescriptiveParallelEvent, which includes DescriptiveStructuredValue because Event does...but Contributor doesn't. Removed.